keycloak/api_client.go

```diff
@@ -54,13 +54,11 @@ func (c *KeycloakClient) post(url string, v interface{}) (string, error) {
 	req.Header.Set("Content-Type", "application/json")
 	resp, err := c.do(req)
 
-	fmt.Println(string(reqBody))
-
 	if err != nil {
 		return "", err
 	}
 
-	if resp.StatusCode != 201 {
+	if resp.StatusCode != 201 && resp.StatusCode != 204 {
```

Comment:
Just a thought for the future, perhaps this should just accept anything >= 200 and <300